datasets.py

```diff
@@ -41,657 +40,77 @@ def clean_ceo_data(df: pd.DataFrame) -> pd.DataFrame:
 
 
 datasets = [
-    CustomLabeledDataset(
```

Comment:
If you keep `openmapflow` at or below 0.1.5, these changes are not necessary for now.
Updating to new version can be done separately

Reply:
So should I revert this to the previous version (within the PR)?